lib/engine.js

```diff
@@ -59,6 +59,13 @@ Engine.prototype.start = function(callback) {
     new MongoDBStorageAdapter(this.storage),
     { disableReaper: true }
   );
+  this.client = require('redis').createClient(this._config.redis);
```

Comment:
Might be helpful to log that we're connecting to redis and show the connection url:port.